grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -236,6 +236,13 @@ class UserService {
         return isSuperuser(currentUser) || (currentUser.getHighestRole(location) >= otherUser.getHighestRole(location))
     }
 
+    Boolean userHasRoles(String userId, Collection roleTypes, String locationId) {
```

Comment:
I see an inconsistency here - you return the method you've created in the domain, but the return types are different - here it is `Boolean` and in the domain it is `boolean` - I think it should be `boolean` here as well.

Reply:
yes, it should be `boolean`